tox.ini

```diff
@@ -169,10 +169,11 @@ deps = {[testenv:flake8]deps}
 commands =
     flake8 examples/src/
     flake8 \
-        # Ingore D103 missing docstring errors in tests (test names should be self-documenting)
+        # Ignore D103 missing docstring errors in tests (test names should be self-documenting)
         # E203 is not PEP8 compliant https://github.com/ambv/black#slices
         # W503 is not PEP8 compliant https://github.com/ambv/black#line-breaks--binary-operators
         --ignore D103,E203,W503 \
+        --per-file-ignores 'examples/test/legacy/v3_default_cmm.py: D205,D400,D401' \
```

Comment:
:100:
Maybe add a short comment explaining why this is excluded ("copy-paste, intentionally not changing stuff")